images/plbase/python-requirements.txt

```diff
@@ -34,6 +34,8 @@ scipy==1.15.2
 seaborn==0.13.2
 sphinx-markdown-builder==0.6.8
 sphinx==8.1.3
+squawk-cli==2.12.0
+sqlfluff==3.4.1
```

Comment:
I'm wary about putting even more stuff into our container images, especially things that we don't want questions to start relying on.

Reply:
I think this a symptom of the fact we don't have any notion of 'dev' dependencies. uv has a nice concept of this. https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies